### ISSUE TITLE
configuration parameters gets ignored if connecting with websockets

### ISSUE THREAD

**Issue by Gioee** [sqlitecloud/sqlitecloud-js#119]
## Current Behavior

configuration parameters gets ignored if connecting with websockets

## Expected Behavior

configuration parameters like maxdata, maxrows, maxrowset, etc. should work

## Steps to Reproduce the Problem

  1. Initialize a connection with websockets with optional parameters

## Environment

- Version: 1.308
- Platform: Mac
- Node.js Version: v20.12.2

**Comment by Gioee** [comment]
This is the method that gets used with tls in the driver [getInitializationCommands](https://github.com/sqlitecloud/sqlitecloud-js/blob/2a549b8d01d0f3ddbce29a0d457c0afb0bce976f/src/drivers/utilities.ts#L42) we should update the driver and the gateway

**Comment by jacobprall** [comment]
What's the status on this?

**Comment by jacobprall** [comment]
“Incoherent” is there a fix?

**Comment by danielebriggi** [comment]
yes, connection should be configurable even when connecting via websocket

**Comment by Gioee** [comment]
@jacobprall I haven't prioritised it as high because it's not a breaking bug. The user can still set these parameters from the connection string or directly in a query. It's certainly something we have to work on because right now it's not that intuitive.
